packages/store-ui/src/atoms/Select/Select.tsx

```diff
@@ -0,0 +1,22 @@
+import type { SelectHTMLAttributes } from 'react'
+import React, { forwardRef } from 'react'
+
+export interface SelectProps extends SelectHTMLAttributes<HTMLSelectElement> {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const Select = forwardRef<HTMLSelectElement, SelectProps>(function Select(
+  { children, testId = 'store-select', ...props },
+  ref
+) {
+  return (
+    <select ref={ref} data-store-select data-testid={testId} {...props}>
+      {children}
```

Comment:
The idea was that the select would be more flexible if receiving `children` instead of a `optionsArray`, as the user could format the labels as it wishes as well as use option groups, although I don't know if this latter feature is useful. I'm fine with checking if the user passed an `optionsArray` and rendering it over `children`, and if options groups aren't all that useful ignoring `children` at all could be the case. What do you think about these scenarios?

Reply:
Oooh! I haven't thought of that! Yeaah, it might not be all that useful to support option groups, but I think we should support them anyway, just to keep things as flexible as possible.

Now I don't think the `optionsArray` is a good idea anymore 😂  It would just be another way of doing the same thing, and would increase the component's API. I think supporting just `children` is fine 😄